ui/js/models/resultsets_store.js

```diff
@@ -794,18 +791,14 @@ treeherder.factory('ThResultSetStore', [
             repoData.loadingStatus.appending = true;
             var isAppend = (repoData.pushes.length > 0);
             var pushes = { results: [] };
-            var loadRepositories = ThRepositoryModel.load({
-                name: repoData.name,
-                watchRepos: true,
-            });
             var loadResultsets = ThResultSetModel.getResultSets(repoData.name,
                                                                 repoData.rsMapOldestTimestamp,
                                                                 count,
                                                                 true,
                                                                 keepFilters)
                     .then((data) => { pushes = data.data; });
 
-            return $q.all([loadRepositories, loadResultsets])
```

Comment:
This was only loaded here because it was a good "first load place" way back when.  We don't even use this model in the ``resutlsets_store``.  :)